src/tests/ftest/container/autotest.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from apricot import TestWithServers
+from command_utils_base import CommandFailure
+
+
+class ContainerAutotestTest(TestWithServers):
+    # pylint: disable=too-few-public-methods
+    """Tests container autotest.
+
+    :avocado: recursive
+    """
+
+    def __init__(self, *args, **kwargs):
+        """Initialize object."""
+        super().__init__(*args, **kwargs)
+        self.daos_cmd = None
+
+    def test_container_autotest(self):
+        """Test container autotest.
+
+        :avocado: tags=all,full_regression,daily,hw,small,quick,autotest
```

Comment:
We should probably use the new tag specification format here:
```suggestion
        :avocado: tags=all,full_regression,daily_regression
        :avocado: tags=hw,small
        :avocado: tags=container,autotest,containerautotest
```

Reply:
Will change